src/stactools/hotosm/cli.py

```diff
@@ -92,15 +123,139 @@ def create_pgstac_from_ctx(
     callback=create_pgstac_from_ctx,
 )
 
+dump_to_path = click.option(
+    "--file",
+    type=click.Path(
+        writable=True,
+        path_type=Path,
+    ),
+    help="Dump STAC Items to this file.",
+    required=True,
+)
+
+
+catalog_option = click.option(
+    "--catalog",
+    type=click.Choice(["OAM", "Maxar"]),
+    required=True,
+    help="Sync STAC Collection definition for this dataset catalog.",
+)
+
 
+# ===== CLI commands
 @click.group
 def main():
     """STAC for Humanitarian OpenStreetMap Team OpenAerialMap."""
 
 
 @main.command()
+@dump_to_path
+@catalog_option
+def dump_collection(
+    file: Path,
+    catalog: str,
+    **_pgstac_options: Any,
+) -> None:
+    """Dump Collection definition to JSON.
+
+    The output of this CLI program can be used with the `pypgstac load collections` CLI
+    command to bulk load STAC Items into PgSTAC.
+    """
+    create_and_save_collection(catalog, file)
```

Comment:
Nit: I generally prefer to print to stdout by default, which enables piping or redirection to a file. But this works fine and matches the use-case (where we need a file for `pypgstac`) so no change needed.